content/docs/user-guide/basic-concepts.md

```diff
@@ -0,0 +1,129 @@
+# Basic Concepts
+
+Intro and DVC philosophy...possible diagram of cache/remote/workspace
```

Comment:
Well, the most basic concept in DVC is that of versioning through codification (currently being worked in #1747). So we could mention that and link to the Versioning use case as an intro.

Reply:
I see this page no longer exists so will resolve. We could consider to use an index page here to address:

> - Consider also a related DVC Principles section (more philosophical, maybe in the Concepts section index)? See https://github.com/iterative/dvc.org/issues/550#issuecomment-544106096 above.

But for now I've scratched out that point from the goals of this PR to improve the scope.